template/src/shared/hooks/useActiveBreakpoint/useActiveBreakpoint.hook.ts

```diff
@@ -1,8 +1,9 @@
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
+
 import { Breakpoint } from '../../../theme/media';
 
 export const useActiveBreakpoint = (): Breakpoint => {
   const theme = useContext(ThemeContext);
-  return theme.activeBreakpoint;
+  return theme.activeBreakpoint || Breakpoint.DESKTOP;
```

Comment:
In this case it won't matter, but generally would be safer to use `theme.activeBreakpoint ?? Breakpoint.DESKTOP` in case if underlying enum value is `0`

Reply:
Sure, I will change it